internal/uvm/start.go

```diff
@@ -205,6 +240,11 @@ func (uvm *UtilityVM) Start(ctx context.Context) (err error) {
 		}
 		uvm.guestCaps = *uvm.gc.Capabilities()
 		uvm.protocol = uvm.gc.Protocol()
+
+		// initial setup required for external GCS connection
+		if err = uvm.configureHvSocketForGCS(ctx); err != nil {
+			return fmt.Errorf("failed to do initial GCS setup: %s", err)
+		}
```

Comment:
Should we be checking that the option for ExternalGuestConnection is set before doing this?

Reply:
Oh never mind. I see that gcListener is only set when opts.ExternalGuestConnection is set as well.